packages/core/lib/src/widgets/tweet/detailed_tweet_metrics.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:core/core.dart';
+import 'package:flutter/material.dart';
+
+/// Likes, Retweets, and Quote Tweet metrics for a detailed tweet
+class DetailedTweetMetrics extends StatelessWidget {
+  /// Creates a new instance of [DetailedTweetMetrics]
+  const DetailedTweetMetrics({
+    super.key,
+    this.retweetsCount = 0,
+    this.quoteTweetsCount = 0,
+    this.likesCount = 0,
+  });
+
```

Comment:
For this Widget, the alignment of numbers and text seems to be a bit off 
![image](https://user-images.githubusercontent.com/16862002/199103506-0d65dd1f-1a88-4f1e-a8e0-fc24e8598c12.png)

Reply:
The line height in this font seems very inconsistent. I set it to 1 for all styles and I think this fixed this widget. However, maybe we can consider a different font if the problem persists?